frontend/src/javascript/stage/Painter.js

```diff
@@ -61,4 +61,28 @@ export class Painter {
     getLayer() {
         return this.layers[this.layerPosition];
     }
-}
+
+    rescalePoints(points) {
+        let minX = Math.min.apply(null, points.map(function(point) { return point.x; }));
+        let maxX = Math.max.apply(null, points.map(function(point) { return point.x; }));
+        let minY = Math.min.apply(null, points.map(function(point) { return point.y; }));
+        let maxY = Math.max.apply(null, points.map(function(point) { return point.y; }));
+
+        let scaleX = 1;
```

Comment:
Trochę też kłuje mnie w oczy, że point to obiekt z polami x i y, ale min,max i scale (x, y) to osobne zmienne.

Reply:
To samo tyczy się `center` (x,y). `Min` i `max` raczej bym zostawił jak jest.